src/stylesheets/components/_header.scss

```diff
@@ -0,0 +1,13 @@
+.header {
+
```

Comment:
@carryall How about to remove this empty space?

Reply:
@junan resolved along with rule adjustment on Stylelint 🙌  [e1e383a](https://github.com/carryall/survey-challenge/pull/23/commits/e1e383a76e16bc7d136f9ade9dbd61ac907d3db2)